setup.py

```diff
@@ -10,8 +10,9 @@
 from distutils.core import setup
 
 setup(name='PrivCount',
-      version='0.2.0',
+      version='1.0.0',
       description='Safely gather and aggregate Tor statistics',
+      url='https://github.com/privcount',
       author='Rob Jansen',
```

Comment:
Sure. Did you want to include minor authors as well?
(Aaron, Nick Hopper, Tariq, Ian Goldberg, ...)

I think it's probably ok to just have them as contributors. Not sure how should track or record that, I think git is sufficient for now.